portfolio/src/main/webapp/map.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDMDY-HirAEOfj9QL7SiprwzK7FvjEw4ro"></script>
```

Comment:
Do your instructions mention anything about how to restrict the use of this key?  I found this: https://developers.google.com/maps/documentation/javascript/get-api-key

Reply:
Thanks, I have restricted the key on the GCP console.